src/include/addr_parsing.h

```diff
@@ -14,6 +14,9 @@
 extern "C" {
 #endif
 
+/*
+* returns a number which allows to know if cat is safe for the memory
```

Comment:
Please check the other files for the conventions regarding the documentation of a function. It uses http://www.stack.nl/~dimitri/doxygen/